src/backend/_scss/application.scss

```diff
@@ -51,25 +51,42 @@ a {
 }
 
 /* Navbar adjustments */
+.logo-container {
+  width: 110px;
+  height: 50px;
+  margin-left: 20px;
+  margin: 1em 0;
+  white-space: nowrap;
+  text-align: center;
+}
+
 .logo {
-  max-height: 45px;
+  position: absolute;
   max-width: 110px;
+  max-height: 50px;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  vertical-align: middle;
+}
+
+.helper {
+  position: relative;
+  display: inline-block;
+  height: 100%;
+  vertical-align: middle;
 }
 
 .navbar-default {
   background-color: $navbar-color;
   border-color: $gray-extra-light;
 
   .navbar-nav > li > a {
-    color: $black;
+    color: $gray-extra-dark;
```

Comment:
@anu-007 please resume the color as it was earlier. Kindly pull the latest code, if it has changed in the previous PR, it will not be reflected as a change in your work.